### PR TITLE
Standardize on new two-line license header

### PR DESCRIPTION
**Pull request by am11**
Cherry-pick of https://github.com/am11/diagnostics/commit/f7f4c566bdd56ae5057352f6abc8b0bbfbb513e8.

### CODE REVIEW

File: eng/InstallRuntimes.proj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- All Rights Reserved. Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<!-- All Rights Reserved. Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
```
Comment (hoyosjs) [inline eng/InstallRuntimes.proj]:
```suggestion
```

File: eng/CreateVersionFile.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!-- All Rights Reserved. Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<!-- All Rights Reserved. Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
```
Comment (hoyosjs) [inline eng/CreateVersionFile.csproj]:
```suggestion
```

File: src/SOS/SOS.UnitTests/Debuggees/DesktopClrHost/DesktopClrHost.cpp [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
+//
```
Comment (hoyosjs) [inline src/SOS/SOS.UnitTests/Debuggees/DesktopClrHost/DesktopClrHost.cpp]:
```suggestion
```

File: src/SOS/Strike/sos_unixexports.src [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ; Licensed to the .NET Foundation under one or more agreements.
 ; The .NET Foundation licenses this file to you under the MIT license.
-; See the LICENSE file in the project root for more information.
+;
```
Comment (hoyosjs) [inline src/SOS/Strike/sos_unixexports.src]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_clru.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_clru.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_clrthreads.py [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_clrthreads.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_clrstack.py [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_clrstack.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpclass.py [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpclass.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpheap.py [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpheap.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpil.py [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpil.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumplog.py [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumplog.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpmd.py [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpmd.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_histobjfind.py [hunk 13]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_histobjfind.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dso.py [hunk 14]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dso.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpobj.py [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpobj.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpmt.py [hunk 16]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpmt.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpmodule.py [hunk 17]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpmodule.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_dumpstack.py [hunk 18]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_dumpstack.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_eeheap.py [hunk 19]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_eeheap.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_eestack.py [hunk 20]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_eestack.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_gcroot.py [hunk 21]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_gcroot.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_histclear.py [hunk 22]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_histclear.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_histinit.py [hunk 23]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_histinit.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_histobj.py [hunk 24]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_histobj.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_histroot.py [hunk 25]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_histroot.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_name2ee.py [hunk 26]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_name2ee.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_ip2md.py [hunk 27]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_ip2md.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/test_libsosplugin.py [hunk 28]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/test_libsosplugin.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/testutils.py [hunk 29]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/testutils.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_bpmd_nofuturemodule_module_function.py [hunk 30]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_bpmd_nofuturemodule_module_function.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_bpmd_module_function_iloffset.py [hunk 31]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_bpmd_module_function_iloffset.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_bpmd_module_function.py [hunk 32]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_bpmd_module_function.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_bpmd_methoddesc.py [hunk 33]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_bpmd_methoddesc.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_bpmd_clearall.py [hunk 34]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_bpmd_clearall.py]:
```suggestion
```

File: src/SOS/lldbplugin.tests/t_cmd_bpmd_clear.py [hunk 35]
```diff
@@ -1,6 +1,6 @@
 # Licensed to the .NET Foundation under one or more agreements.
 # The .NET Foundation licenses this file to you under the MIT license.
-# See the LICENSE file in the project root for more information.
+#
```
Comment (hoyosjs) [inline src/SOS/lldbplugin.tests/t_cmd_bpmd_clear.py]:
```suggestion
```

**Review by hoyosjs — APPROVED** [review]

**Comment by am11** [comment]
cleaned up empty comments with this script:

```sh
git show --name-only --pretty="" |\
  xargs -I{} sh -c \
    'content="$(sed -n 3p $0)"; if [ "$content" = ";" ] || [ "$content" = "//" ] || [ "$content" = "#" ]; then sed -i 3d $0; fi' {}
```

**Review by hoyosjs — APPROVED** [review]

**Comment by hoyosjs** [comment]
Thanks @am11

**Pull request merged**